file_formats/ie_formats/itm_v1.htm

```diff
@@ -571,20 +571,23 @@
           <tr>
             <td>0x000d</td>
             <td>1 (char)</td>
-            <td>Resistance<br />
+            <td>Magic Resistance<br />
```

Comment:
This is incorrect, as there are more internal resistances at play here than just the user visible magic one (bit 2 lists some, but there are also immunities).

Reply:
OK, reverted it back to just "Resistance".

Reply:
ok, hopefully nobody will be nosy about the rest.